rfcs/text/0017-remove-log-original.md

```diff
@@ -18,54 +18,59 @@ The request is to consolidate `log.original` and `event.original` by removing `l
 
 - The internal description of the field `event.original` in [`event`](0017/event.yml) should be updated to reflect the revised scope 
 
-- The [`Beats default fields inclusion list`](../../scripts/generators/beats_default_fields_allowlist.yml) list should be updated by removing `log.original` if/when Deprecation progresses to Removal
+- The extended description of `log.original` in the [`Log Fields documentation`](https://github.com/elastic/ecs/blob/master/docs/field-details.asciidoc#field-log-original) should be amended by addition of a notice of deprecation and subsequently removal if/when Deprecation progresses to Removal
 
-- The extended description of `log.original` in the [`Log Fields documentation`](../../docs/field-details.asciidoc#field-log-original) should be amended by addition of a notice of deprecation and subsequently removal if/when Deprecation progresses to Removal
-
-- The extended description of `event.original` in the [`Event Fields documentation`](../../docs/field-details.asciidoc#field-event-original) should be amended to clarify the absorption of `log.original`
-
-<!--
-Stage 2: Add or update all remaining field definitions. The list should now be exhaustive. The goal here is to validate the technical details of all remaining fields and to provide a basis for releasing these field definitions as beta in the schema. Use GitHub code blocks with yml syntax formatting.
--->
+- The extended description of `event.original` in the [`Event Fields documentation`](https://github.com/elastic/ecs/blob/master/docs/field-details.asciidoc#field-event-original) should be amended to clarify the absorption of `log.original`
 
+ 
 ## Usage
 
-<!--
-Stage 1: Describe at a high-level how these field changes will be used in practice. Real world examples are encouraged. The goal here is to understand how people would leverage these fields to gain insights or solve problems. ~1-3 paragraphs.
--->
+The following examples are taken verbatim from the existing field definitions 
+and are included for completeness.
+
+These are the raw texts of entire events, for example a log message. They 
+differs from the extracted `message` field in that no processing has been 
+applied and the field is not indexed by default. The field can still be 
+retrieved from `_source` and is well-suited to demonstration of log integrity
+or in a re-index pipeline 
 
 ## Source data
 
-<!--
-Stage 1: Provide a high-level description of example sources of data. This does not yet need to be a concrete example of a source document, but instead can simply describe a potential source (e.g. nginx access log). This will ultimately be fleshed out to include literal source examples in a future stage. The goal here is to identify practical sources for these fields in the real world. ~1-3 sentences or unordered list.
--->
+Any or all incoming log or event messages.
```

Comment:
Should we add a condition here since `event.original` will be made optional in many integrations (based on https://github.com/elastic/integrations/issues/777)?

```suggestion
Any or all incoming log or event messages when the original value of the event needs to be preserved.
```